app/mailers/notifier.rb

```diff
@@ -57,7 +57,7 @@ def order_detail_status_changed(order_detail_id)
     @order_detail = OrderDetail.includes(:order, :order_status, :product).find(order_detail_id)
     mail(
       to: @order_detail.order.user.email,
-      subject: "[NUcore #{@order_detail.facility.abbreviation}] Order Status Changed To: #{@order_detail.order_status.name}"
+      subject: "[#{text(:app_name)} #{@order_detail.facility.abbreviation}] Order Status Changed To: #{@order_detail.order_status.name}"
```

Comment:
Would it be worth pulling the whole string out and `text(:subject, facility: @order_detail.facility.abbreviation, status: @order_detail.order_status.name)`?

```
subject: "!app_name! %{facility} Order Status Changed To: %{status}"
```